sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md

```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
 weight: 123
 ---
 
-Pipeline publications should acknowledge the nf-core community (e.g. in an academic journal).
+## Acknowledging nf-core in Academic Publications
 
-**At a minimum**, the nf-core community should be thanked in the acknowledgment section. A good example is from the [nf-core/clipseq](https://doi.org/10.12688/wellcomeopenres.19453.1)) preprint:
+If your work has utilized nf-core pipelines or elements of the framework, please ensure that you cite the following publication, which describes the community and framework in detail:
 
-> We would also like to thank the nf-core team, in particular Harshil Patel, Phil Ewels and Maxime Garcia, for welcoming us to the nf-core community and their assistance in reviewing and releasing the pipeline. We would also like to thank the nf-core community for developing the nf-core infrastructure and resources for Nextflow pipelines. A full list of nf-core community members is available at https://nf-co.re/community.
+> Ewels, P.A., Peltzer, A., Fillinger, S. et al. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol 38, 276–278 (2020). https://doi.org/10.1038/s41587-020-0439-x
 
-Other examples: ([nf-core/eager](https://peerj.com/articles/10947/#acknowledgements)).
+### Minimum Requirement: Acknowledgments Section
 
-**Optionally**, the nf-core community can be included as a consortium co-author of the publication ([example](https://doi.org/10.3390/ijms232314512)). This route is a good idea when a pipeline has used extensive existing nf-core pipeline infrastructure (e.g. modules) that were written by other members of the community not directly involved in the pipeline itself.
+At a minimum, the **nf-core** community should be acknowledged in the **Acknowledgments** section of your paper. Here’s an example from the [nf-core/clipseq preprint](https://doi.org/10.12688/wellcomeopenres.19453.1):
 
-If any members of the nf-core community have provided significant input to the creation of the pipeline, please consider adding them as coauthors on the paper directly.
```

Comment:
I would still have a separate sentence saying you should thank specific people too (and consider the co-author ship) here, I think many won't look at the next two sections if indicated as 'optional'

Reply:
OR ust remove optional from the consortoum thing